HackingWithSwift/SwiftUI/Project1/MyProject/MyProject/ContentView.swift

```diff
@@ -0,0 +1,145 @@
+                                            .fill(Color.white))
+                            
+                            }
+                        }
+                    
+                    VStack (spacing: 1) {
+                            Picker("FROM", selection: $currencyToExchange) {
+                                ForEach (currency, id:\.self) {
+                                    Text($0)
+                                        .onChange(of: currencyToExchange) { newValue in
+                                            convert()
+                                        }
+                                }
+                            }
+                            .frame(maxWidth: .infinity)
+                            .padding(7)
+                            .background(
+                                RoundedRectangle(cornerRadius: 10)
+                                    .fill(Color.white))
+                            
+                        Divider()
+                        
+                            Picker("TO", selection: $currencyExchanged) {
+                                ForEach (currency, id:\.self) {
+                                    Text($0)
+                                        .onChange (of: currencyExchanged) { newValue in
+                                            convert()
+                                            
+                                        }
+                                }
+                            }
+                            .frame(maxWidth: .infinity)
+                            .padding(7)
+                            .background(
+                                RoundedRectangle(cornerRadius: 10)
+                                    .fill(Color.white))
+                            
+                        
+                        
+                    }
+                        VStack {
+                            Text("Conversion")
+                            .font(.subheadline)
+                            .foregroundStyle(Color.gray)
+                            .frame(width: 350, height: 15,alignment: .leading)
+                            Text(convertToAmount, format: .currency(code: (currencyExchanged)))
+                                .frame(width: 350, height: 30,alignment: .leading)
+                                .padding(7)
+                                .background(
+                                    RoundedRectangle(cornerRadius: 10)
+                                        .fill(Color.white))
+                            
+                                
+                                
+                        }
+//                        .pickerStyle(.menu)
+                    }
+             .padding(.horizontal, 24)
+                }
+                .navigationTitle("Exchange 24")
+                
+            }
+            
+        }
+    }
+
+
+#Preview {
+    ContentView()
+}
```

Comment:
Remember to keep your code clean and properly formatted.

Reply:
Also, try splitting your code into smaller pieces using functions or computed properties, such as:

```
func buildPickers() -> some View { 
    // Put your pickers here
} 
```

or 

```
var pickers: some View { 
    // Put your pickers here
} 
```

This will make your code easier to read. **Apply this approach to all your components.**